guides/source/active_support_core_extensions.md

```diff
@@ -946,6 +946,18 @@ A.new.x = 1
 A.new.x # NoMethodError
 ```
 
+A default value can be set using the option `:default`.
+
+```ruby
+module ActiveRecord
+  class Base
+    class_attribute :table_name_prefix, default: "my"
```

Comment:
This seems to subtlety suggest you can set an app-wide table name prefix like this. Let's just replace:

```ruby
module ActiveRecord
   class Base
     class_attribute :table_name_prefix, instance_writer: false, default: ""
   end
 end
```

in the example above. No need to elaborate with any text. `:default` is pretty self-explanatory :)